From e79d56db94497331c6e9a8dd70465e0a55ff1740 Mon Sep 17 00:00:00 2001
From: Samreen Nilofer <samreen.nilofer@intel.com>
Date: Thu, 12 Feb 2015 10:04:06 +0530
Subject: [PATCH 18/24] ASoC: dfw: Add media3_out capture pipeline

Issue: GMINL-4319

Added new media3_out pipeline in media task for capture path.
This pipeline will gets the data from interleaver and will be connected to frontend.
media3_out will be connected to secondary capture path.

Change-Id: I57e615f6c6740fe6da73bacaf11cec736e9568d6
Signed-off-by: Samreen Nilofer <samreen.nilofer@intel.com>
---
 sound/soc/intel/platform-libs/atom_pipes.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/sound/soc/intel/platform-libs/atom_pipes.h b/sound/soc/intel/platform-libs/atom_pipes.h
index c5b445e..9fe301b 100644
--- a/sound/soc/intel/platform-libs/atom_pipes.h
+++ b/sound/soc/intel/platform-libs/atom_pipes.h
@@ -59,6 +59,7 @@ enum sst_dfw_path_index {
 	SST_DFW_PATH_INDEX_MEDIA0_OUT	      = (0x14 << SST_DFW_PATH_ID_SHIFT),
 	SST_DFW_PATH_INDEX_MEDIA1_OUT	      = (0x15 << SST_DFW_PATH_ID_SHIFT),
 	SST_DFW_PATH_INDEX_MEDIA2_OUT	      = (0x16 << SST_DFW_PATH_ID_SHIFT),
+	SST_DFW_PATH_INDEX_MEDIA3_OUT	      = (0x17 << SST_DFW_PATH_ID_SHIFT),
 
 	SST_DFW_PATH_INDEX_FM_OUT	      = (0x18 << SST_DFW_PATH_ID_SHIFT),
 
-- 
1.9.1

